src/styles/blocks/features.scss

```diff
@@ -0,0 +1,65 @@
+.features {
+  &__photos {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
+    margin-bottom: 8px;
+  }
+
+  &__description {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 140%;
+
+    @include onDesktop {
+      width: 672px;
+    }
+  }
+
+  &__image-container {
+    width: 100%;
+    box-sizing: border-box;
+    max-height: 320px;
+    position: relative;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+
+    @include onIpad {
+      height: 186px;
+      padding-bottom: 0;
+
+      &::after {
+        content: "";
+        display: block;
```

Comment:
You don't need to duplicate `content` and `display` rules in media queries if they did not change.